server/src/main/java/dev/slimevr/config/VMCConfig.kt

```diff
@@ -3,7 +3,27 @@ package dev.slimevr.config
 class VMCConfig : // Address of the VRM to be used
 	OSCConfig() {
 	// Anchor the tracking at the hip?
+
+	private var portIn: Int = 39540
+	private var portOut: Int = 39539
+
 	var anchorHip = true
 
 	var vrmJson: String? = null
+
+	override fun getPortIn(): Int {
+		return portIn
+	}
+
+	override fun setPortIn(portIn: Int) {
+		this.portIn = portIn
+	}
+
+	override fun getPortOut(): Int {
+		return portOut
+	}
+
+	override fun setPortOut(portOut: Int) {
+		this.portOut = portOut
+	}
```

Comment:
Can't this just use override properties?Can't these getters and setters just be override properties?
https://kotlinlang.org/docs/inheritance.html#overriding-properties
```suggestion
	override var portIn: Int = 39540
	override var portOut: Int = 39539

	var anchorHip = true

	var vrmJson: String? = null
```

Reply:
Why does VMCConfig overrides all the same from OSCConfig... also just rewrite it all in kotlin without getters and setters.